### PR TITLE
Add Korean keymap

### PR DESCRIPTION
**Pull request by westporch**
Hello. 
I'm newbie.
I've added Korean keymap.
Thanks.

### CODE REVIEW

File: instfiles/Makefile.am [hunk 1]
```diff
@@ -26,7 +26,8 @@ dist_startscript_DATA = \
   km-0000080c.ini \
   km-00000813.ini \
   km-00000816.ini \
-  km-0000100c.ini 
+  km-0000100c.ini \
+  km-00000412.ini
```
Comment (proski) [inline instfiles/Makefile.am]: Let's keep the keymaps sorted alphabetically in Makefile.am

Comment (metalefty) [inline instfiles/Makefile.am]: Let's insert this after `km-00000411.ini` rather than appending to the end.

File: xrdp/xrdp_keyboard.ini [hunk 2]
```diff
@@ -110,3 +112,4 @@ rdp_layout_ch=ch
 rdp_layout_pt=pt
 rdp_layout_br=br(abnt2)
 rdp_layout_pl=pl
+rdp_layout_kr=kr
```
Comment (metalefty) [inline xrdp/xrdp_keyboard.ini]: After jp, too.

**Comment by westporch** [comment]
I sorted alphabetically order.
Thanks :-)

**Comment by metalefty** [comment]
LGTM, thanks. I think we can include this to 0.9.1.

**Comment by proski** [comment]
I agree, this can go to 0.9.1

**Pull request merged**